src/ddmd/func.d

```diff
@@ -2286,12 +2293,15 @@ extern (C++) class FuncDeclaration : Declaration
      * Find index of function in vtbl[0..dim] that
      * this function overrides.
      * Prefer an exact match to a covariant one.
+     * Params:
+     *      fix17349 = enable fix https://issues.dlang.org/show_bug.cgi?id=17349
      * Returns:
      *      -1      didn't find one
      *      -2      can't determine because of forward references
      */
-    final int findVtblIndex(Dsymbols* vtbl, int dim)
+    final int findVtblIndex(Dsymbols* vtbl, int dim, bool fix17349 = true)
```

Comment:
I don't think the longer name adds much. There's a clickable link in the description of the parameter that should suffice. I think the dmd source code in general could benefit more from clickable links in the comments :-)